ts/client/src/accounts/group.ts

```diff
@@ -437,13 +437,19 @@ export class Group {
         }),
       );
     } else {
+      const pKs = Array.from(this.openbookV2MarketsMapByExternal.values()).map(
+        (m) => m.openbookMarketExternal,
```

Comment:
minor: why not use keys?

Reply:
could do, but they are strings so would have to map anyway to get pubkeys for the rpc call method